src/components/FeatureAttributes/fields/FeatureAttributeIdFeatureField/FeatureAttributeIdFeatureField.tsx

```diff
@@ -32,7 +34,8 @@ export const FeatureAttributeIdFeatureField: FC<
   if (
     !featureType ||
     !allowedFeatureTypes.includes(featureType) ||
-    !allowedDataTypes.includes(dataType)
+    !allowedDataTypes.includes(dataType) ||
+    !purposes.includes("synthesis")
```

Comment:
An ID feature is required for timeseries, which is not limited to synthesis

Reply:
Marked IdFeature as a core field.